Packs/CarbonBlackEnterpriseEDR/TestPlaybooks/Carbon_Black_Enterprise_EDR_Test.yml

```diff
@@ -260,7 +265,7 @@ tasks:
       - "7"
     scriptarguments:
       watchlist_id:
-        simple: ${CarbonBlackEEDR.Watchlist.ID}
+        simple: ${watchListId}
```

Comment:
Due to not wanting to break BC for keys in different case (upper/ lower) to avoid duplictions.